### PR TITLE
Update invoker.py

### PR DESCRIPTION
**Pull request by ghost**
Fix for error: WARNING:urllib3.connectionpool:Connection pool is full, discarding connection

### CODE REVIEW

File: aws_inventory/invoker.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def _probe_services(self):
             # create a config for all clients to share
             client_config = botocore.config.Config(
                 connect_timeout=config.CLIENT_CONNECT_TIMEOUT,
+                max_pool_connections=25,
```
Comment (bitsandsalsa) [inline aws_inventory/invoker.py:52]: Why 25? Is there a way to derive a good value based off a client's runtime environment?

Comment (aleon1220) [reply aws_inventory/invoker.py:52]: It should be added to `config.py` first then set in the `invoker.py` as

``` python
max_pool_connections=config.CLIENT_MAX_POOL_CONNECTIONS
```

**Comment by aleon1220** [comment]
@bitsandsalsa could you review this PR please to fix the max_pool_connections warning?
thanks

**Comment by bitsandsalsa** [comment]
I took @aleon1220 suggestion. Thank you @BlueCanary-DM for your contribution.

**Pull request merged**